src/hotspot/share/opto/divnode.cpp

```diff
@@ -1379,37 +1397,49 @@ const Type* ModLNode::Value(PhaseGVN* phase) const {
 
 
 //=============================================================================
-//------------------------------Value------------------------------------------
-const Type* ModFNode::Value(PhaseGVN* phase) const {
-  // Either input is TOP ==> the result is TOP
-  const Type *t1 = phase->type( in(1) );
-  const Type *t2 = phase->type( in(2) );
-  if( t1 == Type::TOP ) return Type::TOP;
-  if( t2 == Type::TOP ) return Type::TOP;
+//------------------------------Idealize---------------------------------------
```

Comment:
I would just remove these lines if you are already toughing the code here:
`//------------------------------Idealize---------------------------------------`